packages/sdks/nextjs-sdk/package.json

```diff
@@ -140,7 +140,7 @@
 	},
 	"peerDependencies": {
 		"@types/react": ">=18",
-		"next": ">=14.2.21",
+		"next": ">=15.2.3",
```

Comment:
@omercnet this impacts developers using nextjs sdk as well, I would not change it so fast

Reply:
update to 14.2.15 then? so not to require 15 but still avoid the CVE